core/scheduler/metrics.go

```diff
@@ -45,13 +45,20 @@ var (
 		Help:      "The total count of duties scheduled by type",
 	}, []string{"duty"})
 
-	activeGauge = promauto.NewGauge(prometheus.GaugeOpts{
+	activeValsGauge = promauto.NewGauge(prometheus.GaugeOpts{
 		Namespace: "core",
 		Subsystem: "scheduler",
 		Name:      "validators_active",
 		Help:      "Number of active validators",
 	})
 
+	totalValsGauge = promauto.NewGauge(prometheus.GaugeOpts{ //nolint:promlinter
```

Comment:
the metric is `core_scheduler_validators_total` and the promlinter complains that "non-counter metrics should not have `_total` suffix"